tools/phenomenal/isatools/create_metabo/isa_create_metabo.xml

```diff
@@ -7,21 +7,22 @@
 
     <requirements>
         <requirement type="package" version="6.7">click</requirement>
-        <requirement type="package" version="0.9.4">isatools</requirement>
+        <!--<requirement type="package" version="0.9.4">isatools</requirement>-->
```

Comment:
At least for the dynamic destination, the requirement is not a problem, in case you were removing it because of that.

Reply:
Actually it's commented out in there because in the upstream `isatools-galaxy` project where the tool XMLs live together, I've got it to install via pip rather than conda temporarily until bioconda updates our `isatools` recipe...